students/drageon/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import UserListView
+
+urlpatterns = [
+    path("/us", UserListView.as_view())
```

Comment:
URL을 `:8000/user/signup`으로 만들어주세요~
```suggestion
    path("/signup", UserListView.as_view())
```